src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export const enum Status {
+  idle,
+  loading,
+  failed,
```

Comment:
enum's items should be in CamelCase or in UPPER_CASE

Reply:
Done